deployments/stacks/dpe-k8s/main.tf

```diff
@@ -43,3 +43,15 @@ module "sage-aws-ses" {
 
   email_identities = var.ses_email_identities
 }
+
+module "synapse_dataset_to_crossiant_metadata" {
+  # This points to the prod cluster - The only location where we want to create the bucket
+  count  = var.cluster_name == "dpe-k8" ? 1 : 0
+  source      = "../../../modules/s3-bucket"
+  bucket_name = "synapse-croissant-metadata"
+  enable_versioning = true
+  aws_account_id = var.aws_account_id
+  cluster_name = var.cluster_name
+  cluster_oidc_provider_arn = module.sage-aws-eks.cluster_oidc_provider_arn
+  public_access = false

```

Comment:
@ConsoleCatzirl @zaro0508 Before I flip on this `public_access` boolean flag do you mind giving these changes a review?

See [this design doc](https://sagebionetworks.jira.com/wiki/spaces/DPE/pages/4006379521/DPE-1234+Automate+Synapse+Dataset+to+Croissant+JSON-LD+Files) on what these changes are for.

I will be deploying this out to the `org-sagebase-dpe-prod` AWS account, and once these changes are reviewed I'll create the bucket and make it public.

This ties into changes [found here](https://github.com/Sage-Bionetworks-Workflows/orca-recipes/pull/77) which is the orchestration tool (Apache Airflow) that will run the python script to populate this S3 bucket.